### DISCUSSION TITLE
Templimiter switching on/off continuously when hwc storage is enabled

### DISCUSSION THREAD

**Discussion by Przemko92** [john30/ebusd#1156] [Q&A]
I have strange behaviour of templimiter sensor which is continuously switching on and off whend hwc load is enabled. The parameter can be changed every one second or sometimes it stays in constant state for several hours.
Templimiter prevents my heater to turn on the flame for central heating. Hot water works fine. Which temperature sensor is responsible for this parameter? Is any other reason pf enabling and dosabling templimiter?

![Screenshot_20240119_215504_Home Assistant](https://github.com/john30/ebusd/assets/33545571/21e6fe02-fe81-44a9-91ad-0db6a2156a31)
![Screenshot_20240119_214936_Home Assistant](https://github.com/john30/ebusd/assets/33545571/649e735e-59a5-4dfd-9bf1-5d0cd3132e14)

**Comment by makadizsolt** [comment]
Are you sure that this parameter prevents central heating? My Saunier Duval Thema Condens F/AS25 is heating well ( I mean central heating also, not just the hot water storage) despite this parameter being "on". And this parameter is "on" quite a lot by the way, see this image:

<img width="612" height="187" alt="image" src="https://github.com/user-attachments/assets/b25d78f2-8bdd-4a54-9f98-1c4e42278698" />

Right now this parameter is on, and the boiler heats the radiators (checked) with the flame level being 2 out of 5.

**Reply by Przemko92** [reply]
Does it heat while "off"?

**Comment by makadizsolt** [comment]
Sure, as seen from the pictures below, heating turned on 49 minutes ago, while templimiter only 43 minutes ago, so it was definitely heating while the limiter was off.  The pumpstate "on" means central heating, and "hwc" means hot water storage tank heating.

<img width="617" height="638" alt="image" src="https://github.com/user-attachments/assets/825e3755-78fb-470f-80e3-531969b54c9f" />

<img width="617" height="633" alt="image" src="https://github.com/user-attachments/assets/f3563631-1873-4b96-b0d6-142c158941b1" />
